themes/default/content/blog/converting-bicep-to-pulumi/index.md

```diff
@@ -0,0 +1,357 @@
+---
+title: "Converting Bicep code to Pulumi"
+date: 2023-09-27
```

Comment:
Need to adjust date for actual publish date. Let's try to get it published Friday.

```suggestion
date: 2023-09-29
```